dev-tools/omdb/src/bin/omdb/crucible_agent.rs

```diff
@@ -0,0 +1,177 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+//! omdb commands that query a crucible-agent
+
+use anyhow::bail;
+use anyhow::Context;
+use clap::Args;
+use clap::Subcommand;
+use crucible_agent_client::types::RegionId;
+use crucible_agent_client::Client;
+use tabled::Tabled;
+
+use crate::Omdb;
+
+/// Arguments to the "omdb crucible-agent" subcommand
+#[derive(Debug, Args)]
+pub struct CrucibleAgentArgs {
+    /// URL of the crucible agnet internal API
+    #[clap(long, env("OMDB_CRUCIBLE_AGENT_URL"))]
+    crucible_agent_url: Option<String>,
+
+    #[command(subcommand)]
+    command: CrucibleAgentCommands,
+}
+
+/// Subcommands for the "omdb crucible-agent" subcommand
+#[derive(Debug, Subcommand)]
+enum CrucibleAgentCommands {
+    /// print information about regions
+    #[clap(subcommand)]
+    Regions(RegionCommands),
+    /// print information about snapshots
+    #[clap(subcommand)]
+    Snapshots(SnapshotCommands),
+}
+
+#[derive(Debug, Subcommand)]
+enum RegionCommands {
+    /// Print list of all running control plane regions
+    List,
+}
+
+#[derive(Debug, Subcommand)]
+enum SnapshotCommands {
+    /// Print list of all running control plane regions
+    List,
+}
+
+impl CrucibleAgentArgs {
+    /// Run a `omdb crucible-agent` subcommand.
+    pub(crate) async fn run_cmd(
+        &self,
+        _omdb: &Omdb,
+    ) -> Result<(), anyhow::Error> {
+        // The crucible agent URL is required, but can come
+        // from the environment, in which case it won't be on the command line.
+        let Some(crucible_agent_url) = &self.crucible_agent_url else {
+            bail!(
+                "crucible agent URL must be specified with \
+                --crucible-agent-url or OMDB_CRUCIBLE_AGENT_URL"
```

Comment:
It's clear to me that this is referring to an environment variable, but how about changing the text to something like "or by setting the environment variable OMDB_CRUCIBLE_AGENT_URL" ?

Reply:
Done